float32 x
float32 y
float32 yaw
float32 v_imu_x
float32 v_imu_y
float64 stamp
